client/components/tinymce/plugins/insert-menu/plugin.jsx

```diff
@@ -26,11 +26,28 @@ const initialize = editor => {
 		cmd: menuItems[ 0 ].cmd,
 		menu: menuItems.map( ( { name } ) => editor.menuItems[ name ] ),
 		onPostRender() {
-			ReactDOM.render(
-				<Gridicon icon="add-outline" />,
-				this.$el[ 0 ].children[ 0 ]
-			);
-		}
+			ReactDOM.render( <Gridicon icon="add-outline" />, this.$el[ 0 ].children[ 0 ] );
```

Comment:
I didn't add this line. :) Feel free to open an issue so it can be worked in another PR.